### PR TITLE
 - Fixed issues and Changes in FAQ.

### PR DESCRIPTION
**Pull request by RajkiranBagal**
### Description
Trello Task - https://trello.com/c/j6fryUee/12-new-block-faq-schema

 - When user applied separator,  there is by default boundary around answer block - [Video](https://www.awesomescreenshot.com/video/168402?key=7515551d299b8674590b9c2187ce1f81)
 - Question Color applied to the icon but the active color is not applicable
- Added Equal Height option for Grid layout

### Types of changes
 What types of changes does your code introduce?  
 Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
Fixed issues.

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: classes/class-uagb-block-helper.php [hunk 1]
```diff
@@ -4355,9 +4355,12 @@ public static function get_faq_css( $attr, $id ) {
 			if ( true === $attr['enableSeparator'] ) {
 
 				$selectors[' .uagb-faq-child__outer-wrap .uagb-faq-content '] = array(
-					'border-style'     => 'solid',
-					'border-top-color' => $attr['borderColor'],
-					'border-top-width' => UAGB_Helper::get_css_value( $attr['borderWidth'], 'px' ),
+					'border-style'        => 'solid',
+					'border-top-color'    => $attr['borderColor'],
+					'border-top-width'    => UAGB_Helper::get_css_value( $attr['borderWidth'], 'px' ),
+					'border-right-width'  => UAGB_Helper::get_css_value( 0, 'px' ),
```
Comment (vrundakansara) [inline classes/class-uagb-block-helper.php:4361]: Just add `0`, instead of calling the entire function.

File: src/blocks/faq/edit.js [hunk 2]
```diff
@@ -868,7 +876,8 @@ class UAGBFaqEdit extends Component {
 					`uagb-faq-icon-${ this.props.attributes.iconAlign }`,
 					`uagb-faq-layout-${ this.props.attributes.layout }`,
 					`uagb-faq-expand-first-${ this.props.attributes.expandFirstItem }`,
-					`uagb-faq-inactive-other-${ this.props.attributes.inactiveOtherItems }`
+					`uagb-faq-inactive-other-${ this.props.attributes.inactiveOtherItems }`,
+					`uagb-faq-equal-height-${ this.props.attributes.equalHeight }`
```
Comment (vrundakansara) [inline src/blocks/faq/edit.js:880]: `uagb-faq-equal-height-true` doesn't make sense to me. Use this class conditionally.
```js
const equalHeightClass = equalHeight ? "uagb-faq-equal-height" : ""
```

Comment (RajkiranBagal) [reply src/blocks/faq/edit.js]: Changed.

File: classes/class-uagb-block-helper.php (hunk 1 above)
Comment (RajkiranBagal) [reply classes/class-uagb-block-helper.php]: Done

**Pull request merged**